tests/flow/intersection/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -79,7 +79,8 @@ function hasObjectMode_ok(options: DuplexStreamOptions): boolean {
  * @flow
  */
 
-type DuplexStreamOptions = ReadableStreamOptions & WritableStreamOptions & {
+type DuplexStreamOptions = ReadableStreamOptions &
+  WritableStreamOptions & {
   allowHalfOpen?: boolean,
   readableObjectMode?: boolean,
```

Comment:
Oh you are right, it's missing one level of indentation. Hmm, I need to think about it.